CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPlugin.java

```diff
@@ -184,7 +184,6 @@ public PluginParameters createParameters() {
         final PluginParameter<StringParameterValue> string1 = StringParameterType.build(TEST1_PARAMETER_ID);
         string1.setName("Test1");
         string1.setDescription("A test string");
-//        string1.setStringValue("Plugh.");
```

Comment:
@mimosa2 so these set*Value calls was in the original plugin which I think you commented for testing and they are now removed. Are you able to restore the set value calls?

Reply:
Restored the set value calls.